backend/node/base.py

```diff
@@ -178,8 +178,13 @@ async def get_contract_data(
         from_address: str,
         calldata: bytes,
     ) -> Receipt:
+        now = datetime.now().strftime("%Y-%m-%d %H:%M:%S.%f")
```

Comment:
why not `datetime.now().isoformat()` ?

Reply:
It is also incorrect as it passes current time without timezone info. it should be `datetime.now().astimezone(datetime.timezone.utc).isoformat()`